index.html

```diff
@@ -1,382 +1,382 @@
+                        </div>
+                            <div class="col-md-4 col-sm-6 sponsor-column">
+                            <div class="sponsor text-center">
+                                <a href="http://microsoft.com/" target="default">
+                                    <img alt="Sponsor" src="img/microsoft.png">
+                                </a>
+                            </div>
+                        </div>
+                        <div class="col-md-4 col-sm-6 sponsor-column">
+                            <div class="sponsor text-center">
+                                <a href="http://nextcloud.com/" target="default">
+                                    <img alt="Sponsor" src="img/Nextcloud.png">
+                                </a>
+                            </div>
+                        </div>
 
-			<section class="sponsors">
-				<div class="container">
-					<div class="row">
-						<div class="col-sm-12 text-center">
-							<h1>Proudly supported by</h1>
-								</div>
-							</div>								
-								<div class="col-md-4 col-sm-6 sponsor-column">
-					<!--			<div class="sponsor text-center">
-									<a href="https://www.google.com/cloud/" target="default">
-										<img alt="Sponsor" src="img/google.png">
-									</a>
-								</div>
-							</div>
-								<div class="col-md-4 col-sm-6 sponsor-column">
-								<div class="sponsor text-center">
-									<a href="https://daimler.com/" target="default">
-										<img alt="Sponsor" src="img/Daimler_logo_gray.png">
-									</a>
-								</div>
-							</div>
-								<div class="col-md-4 col-sm-6 sponsor-column">
-								<div class="sponsor text-center">
-									<a href="http://microsoft.com/" target="default">
-										<img alt="Sponsor" src="img/microsoft.png">
-									</a>
-								</div>
-							</div>
-							<div class="col-md-4 col-sm-6 sponsor-column">
-								<div class="sponsor text-center">
-									<a href="http://nextcloud.com/" target="default">
-										<img alt="Sponsor" src="img/Nextcloud.png">
-									</a>
-								</div>
-							</div>
+                        <div class="col-md-4 col-sm-6 sponsor-column">
+                            <div class="sponsor text-center">
+                                <a href="https://www.opensuse.org" target="default">
+                                    <img alt="Sponsor" src="img/opensuse.png">
+                                </a>
+                            </div>
+                        </div>
 
-							<div class="col-md-4 col-sm-6 sponsor-column">
-								<div class="sponsor text-center">
-									<a href="https://www.opensuse.org" target="default">
-										<img alt="Sponsor" src="img/opensuse.png">
-									</a>
-								</div>
-							</div>
+                        <div class="col-md-4 col-sm-6 sponsor-column">
+                            <div class="sponsor text-center">
+                                <a href="http://intel.com" target="default">
```

Comment:
You are a skilled programmer, I do not doubt that. However, now I see a huge diff for the index file which I am not going to agree to because I like to follow the [CCCC](https://rfc.zeromq.org/spec:22/C4/).

> A patch SHOULD be a minimal and accurate answer to exactly one identified and agreed problem.

The changing the spaces is not an agreed problem.
The patch is not a minimal solution to **one** problem.

There are two solutions for this:
1. I merge and revert.
2. You change the patch.

Now, because I value what you do, I will tell you about my reasoning behind following CCCC:
Building open-source software is not about building software right but about building the right software.
In order to build the right software, we need a lot of ideas from as many people as possible, including you. However, there is a tradeoff. If a change scares away more people than it brings to the software, the diversity and future of the project is at risk of decline.

Thus, I am not against any changes you make. I will gladly accept any changes as long as I think they go by the rules I follow. In your case, 2-3 PRs would do it, I guess.
Also, I am not a gatekeeper. If you find any other person with access rights to merge this, they can do it.

What do you think about this?
How would you like to proceed?

Reply:
@niccokunzmann 
please help me with it..
what i have to do for moving further??

thanks in advance